### ISSUE TITLE
Seperators are strict for extractLanguageDisplayNames

### ISSUE THREAD

**Issue by iwatakeshi** [papandreou/node-cldr#22]
Hi, I'm trying to retrieve the display name for the locale `en-us` but the function returns `{}`. If I normalize the locale to `en_us` then it will return properly. According to the readme, any function that takes a localeId should be able to accept `-` or `_` correct?

I also found a weird bug (?) for `en-gb` where it only returns 
```js
{ 
  arn: 'Araucanian',
  bax: 'Bamum',
  en_gb: 'UK English',
  en_us: 'US English',
  tzm: 'Central Morocco Tamazight',
  wal: 'Walamo' 
}
```
yet normalizing the locale returns 
```js

{ arn: 'Araucanian',
  bax: 'Bamum',
  en_gb: 'UK English',
  en_us: 'US English',
  tzm: 'Central Morocco Tamazight',
  wal: 'Walamo',
  aa: 'Afar',
  ab: 'Abkhazian',
  ace: 'Achinese',
  ach: 'Acoli',
  ada: 'Adangme',
  ady: 'Adyghe',
  //...
  }
```

![cldr](https://cloud.githubusercontent.com/assets/1505448/6732456/92adb5ee-ce11-11e4-80c2-8ddb9c69e41e.PNG)

**Comment by papandreou** [comment]
Confirmed. The display names are actually defined in the `en` locale, but it looks like they don't get inherited properly.

**Comment by papandreou** [comment]
Fixed in 2.3.4. Thanks for reporting!

**Comment by iwatakeshi** [comment]
:+1:
